tests/app/main/test_briefs.py

```diff
@@ -612,8 +612,34 @@ def test_day_rate_question_does_not_replay_buyers_budget_range_if_not_provided(s
         data = res.get_data(as_text=True)
         assert "Buyer's maximum day rate:" not in data
 
-    def test_start_date_question_escapes_brief_start_date_markdown(self):
+    def test_availability_question_shows_correct_content_for_lot(self):
+        self.brief['briefs']['startDate'] = '17/01/2017'
+        self.brief['briefs']['researchDates'] = '17/01/2017'
+        lots = ['digital-specialists', 'digital-outcomes', 'user-research-participants']
+        question_content = ['the specialist can start work?', 'the team can start?', 'you can recruit participants?']
+        hint_content = ['the specialist to start by', 'the team to start by', 'participants:']
+
+        for lot, question, hint in zip(lots, question_content, hint_content):
+            self.brief['briefs']['lot'] = lot
+
+            res = self.client.get(
+                '/suppliers/opportunities/1234/responses/5/availability'
+            )
+            assert res.status_code == 200
+
+            doc = html.fromstring(res.get_data(as_text=True))
+
+            page_heading = doc.xpath("//h1/text()")
+            page_hint = doc.xpath("//span[@id='input-availability-question-advice']/p/text()")
+
+            assert len(page_heading) == 1
+            assert page_heading[0].strip() == 'When is the earliest {}'.format(question)
+            assert len(page_hint) == 1
+            assert page_hint[0] == 'The buyer needs {} 17/01/2017.'.format(hint)
+
+    def test_availability_question_escapes_brief_start_date_markdown(self):
         self.brief['briefs']['startDate'] = '**markdown**'
+        self.brief['briefs']['lot'] = 'digital-specialists'
```

Comment:
I initially questioned why this was needed as we don't need to define it for other tests. As per my comment on the frameworks repo (read this first), other tests run off the `lotSlug` behaviour which is included in the self.brief API stub. If we change this functionality to run off `lotSlug` for consistency then we won't need this line.